src/index.d.ts

```diff
@@ -204,6 +210,11 @@ declare namespace electronPackager {
     afterCopyExtraResources?: HookFunction[];
     /** Functions to be called after the prebuilt Electron binary has been extracted to a temporary directory. */
     afterExtract?: HookFunction[];
+    /**
+     * Functions to be called after the final matrix of platform/arch combination is determined.  Use this to
+     * learn what archs/platforms packager is targetting when you pass "all" as a value.
```

Comment:
> learn what archs/platforms packager is targetting when you pass "all" as a value.

passing "all" as a value will return the architectures/platforms that packager is currently targetting.